components/feature/downloads/src/main/res/values/strings.xml

```diff
@@ -25,6 +25,9 @@
     <!-- Message that appears when the downloaded file could not be opened-->
     <string name="mozac_feature_downloads_could_not_open_file">Could not open file</string>
 
+    <!-- Message that appears when the downloaded file is a Zip file and Android doesn't support opening-->
+    <string name="mozac_feature_downloads_could_not_open_zip">Opening ZIP files not supported</string>
```

Comment:
What do you think about just having one message and replacing **ZIP** for the extension of the not supported file?